api/integrations/github/client.py

```diff
@@ -12,3 +54,154 @@ def generate_token(installation_id: str, app_id: int) -> str:
     Github(auth=auth)
     token = auth.token
     return token
+
+
+# TODO: Add coverage tests for this function
+def generate_jwt_token(app_id: int) -> str:  # pragma: no cover
+    github_auth: Auth.AppAuth = Auth.AppAuth(
+        app_id=app_id,
+        private_key=settings.GITHUB_PEM,
+    )
+    token = github_auth.create_jwt()
+    return token
+
+
+def post_comment_to_github(
+    installation_id: str, owner: str, repo: str, issue: str, body: str
+) -> typing.Dict[str, typing.Any]:
+    token = generate_token(
+        installation_id,
+        settings.GITHUB_APP_ID,
+    )
+
+    url = f"{GITHUB_API_URL}repos/{owner}/{repo}/issues/{issue}/comments"
+    headers = {
+        "Accept": "application/vnd.github.v3+json",
+        "Authorization": f"Bearer {token}",
+    }
```

Comment:
Can we use `build_request_headers` here?

Reply:
Done